corehq/apps/hqwebapp/utils/bootstrap/reports/debug.py

```diff
@@ -0,0 +1,93 @@
+from corehq.apps.hqwebapp.utils.bootstrap.paths import is_bootstrap3_path, get_bootstrap5_path
+from corehq.apps.hqwebapp.utils.bootstrap.reports.progress import (
+    get_migrated_filters,
+    get_migrated_filter_templates,
+)
+
+REPORT_TEMPLATE_PROPERTIES = [
+    ('template_base', 'base_template'),
+    ('template_async_base', 'base_template_async'),
+    ('template_report', 'report_template_path'),
+    ('template_report_partial', 'report_partial_path'),
+    ('template_filters', 'base_template_filters'),
+]
+COMMON_REPORT_TEMPLATES = [
+    "reports/async/bootstrap3/default.html",
+    "reports/bootstrap3/base_template.html",
+    "reports/standard/bootstrap3/base_template.html",
+    "reports/bootstrap3/tabular.html",
+]
+
+
+class Color:
+    CYAN = '\033[96m'
+    GREEN = '\033[92m'
+    WARNING = '\033[93m'
+    BOLD = '\033[1m'
+    ENDC = '\033[0m'
+
+
+def _name(instance):
+    return instance.__class__.__name__
+
+
+def reports_bootstrap5_template_debugger(report_instance):
```

Comment:
example output of what it looks like in the console:

![Screenshot 2025-01-21 at 3 25 59 PM](https://github.com/user-attachments/assets/dc5b8bce-6a3b-4c43-9aa1-2733ad55e640)
![Screenshot 2025-01-21 at 3 24 51 PM](https://github.com/user-attachments/assets/0a17d7f8-c38a-41f1-b544-723e94201978)